questionnaire-ko.json

```diff
@@ -1,434 +1,434 @@
-{
-  "specVersion": "2.1.1",
-  "language": "ko",
-  "title": "Conformance in Questions",
-  "contextText": "Context",
-  "questionsText": "Questions",
-  "completedText": "Completed?",
-  "statusText": "Status",
-  "answerColumnText": "Answer",
-  "sectionColumnText": "Section",
-  "numberColumnText": "Number",
-  "specRefColumnText": "Spec Ref",
-  "questionColumnText": "Question Text",
-  "tocText": "Table of Contents",
-  "preambleText": "The OpenChain Project builds trust in open source by making license compliance simpler and more consistent. It maintains the OpenChain Specification to define a core set of requirements every quality compliance program must satisfy. The result is that open source license compliance becomes more predictable, understandable and efficient for participants of the software supply chain.\nThis document contains a series of questions to determine whether a company has an OpenChain Conformant program. If each of these questions can be answered with a \"yes\" then that company meets all the requirements of conformance to the OpenChain Specification version 2.1 (functionally identical to ISO/IEC DIS 5230:2020(e)). If any of the questions are answered with a \"no\" then the company can clearly identify where additional investment is needed to improve the compliance process.",
-  "sections": [
-    {
-      "name": "Part 1",
-      "title": "1. Program foundation",
-      "questions": [
-        {
-          "correctAnswer": "Yes",
-          "question": "Do you have a documented policy governing the open source license compliance of the Supplied Software?",
-          "number": "1.a",
-          "type": "YES_NO",
+{
+  "specVersion": "2.1.1",
+  "language": "ko",
+  "title": "준수 확인을 위한 질문",
+  "contextText": "배경",
+  "questionsText": "질문",
+  "completedText": "완료?",
+  "statusText": "상태",
+  "answerColumnText": "답변",
+  "sectionColumnText": "섹션",
+  "numberColumnText": "No",
+  "specRefColumnText": "설명서 Ref",
+  "questionColumnText": "질문 내용",
+  "tocText": "목차",
+  "preambleText": "OpenChain 프로젝트는 오픈소스 라이선스 컴플라이언스를 보다 간단하고 일관성있게 만들어 오픈소스에 대한 신뢰를 구축합니다. OpenChain 설명서는 모든 우수한 컴플라이언스 프로그램이 충족해야하는 핵심 요건을 정의합니다. 그 결과 소프트웨어 공급망의 참가자가 오픈소스 라이선스 컴플라이언스를 보다 예측 가능하고 이해하기 쉽고 효율적으로 만들 수 있습니다.\n이 문서에는 회사가 OpenChain을 준수하는지 여부를 확인하기위한 일련의 질문이 포함되어 있습니다. 이러한 각 질문에 \"예\"라고 대답 할 수 있으면 해당 회사는 OpenChain 설명서 버전 2.0에 대한 모든 요건을 충족합니다. 질문에 \"아니오\"라고 대답하면 회사는 컴플라이언스 프로세스를 개선하기 위해 추가 투자가 필요한 위치를 명확하게 식별 할 수 있습니다.",
+  "sections": [
+    {
+      "name": "Part 1",
+      "title": "1. 프로그램 설립",
+      "questions": [
+        {
+          "correctAnswer": "네",

```

Comment:
The correctAnswer should not be translated

Reply:
Done locally. Pushing later.